### ISSUE TITLE
'Select' is used in 'Drawer'，Option to disappear。

### ISSUE THREAD

**Issue by lqsong** [element-plus/element-plus#789]
```vue
<template>
<el-button type="text" @click="table = true">打开嵌套表格的 Drawer</el-button>
<el-button type="text" @click="dialog = true">打开嵌套 Form 的 Drawer</el-button>
<el-drawer
  title="我嵌套了表格!"
  v-model="table"
  direction="rtl"
  size="50%">
   <el-table :data="gridData">
      <el-table-column property="date" label="日期" width="150"></el-table-column>
      <el-table-column property="name" label="姓名" width="200"></el-table-column>
      <el-table-column property="address" label="地址"></el-table-column>
    </el-table>
</el-drawer>

<el-drawer
  title="我嵌套了 Form !"
  :before-close="handleClose"
  v-model="dialog"
  direction="ltr"
  custom-class="demo-drawer"
  ref="drawer"
  >
  <div class="demo-drawer__content">
    <el-form :model="form">
      <el-form-item label="活动名称" :label-width="formLabelWidth">
        <el-input v-model="form.name" autocomplete="off"></el-input>
      </el-form-item>
      <el-form-item label="活动区域" :label-width="formLabelWidth">
        <el-select v-model="form.region" placeholder="请选择活动区域">
          <el-option label="区域一" value="shanghai"></el-option>
          <el-option label="区域二" value="beijing"></el-option>
        </el-select>
      </el-form-item>
    </el-form>
    <div class="demo-drawer__footer">
      <el-button @click="cancelForm">取 消</el-button>
      <el-button type="primary" @click="$refs.drawer.closeDrawer()" :loading="loading">{{ loading ? '提交中 ...' : '确 定' }}</el-button>
    </div>
  </div>
</el-drawer>
</template>
<script>
export default {
  data() {
    return {
      table: false,
      dialog: false,
      loading: false,
      gridData: [{
        date: '2016-05-02',
        name: '王小虎',
        address: '上海市普陀区金沙江路 1518 弄'
      }, {
        date: '2016-05-04',
        name: '王小虎',
        address: '上海市普陀区金沙江路 1518 弄'
      }, {
        date: '2016-05-01',
        name: '王小虎',
        address: '上海市普陀区金沙江路 1518 弄'
      }, {
        date: '2016-05-03',
        name: '王小虎',
        address: '上海市普陀区金沙江路 1518 弄'
      }],
      form: {
        name: '',
        region: '',
        date1: '',
        date2: '',
        delivery: false,
        type: [],
        resource: '',
        desc: ''
      },
      formLabelWidth: '80px',
      timer: null,
    };
  },
  methods: {
    handleClose(done) {
      if (this.loading) {
        return;
      }
      this.$confirm('确定要提交表单吗？')
        .then(_ => {
          this.loading = true;
          this.timer = setTimeout(() => {
            done();
            // 动画关闭需要一定的时间
            setTimeout(() => {
              this.loading = false;
            }, 400);
          }, 2000);
        })
        .catch(_ => {});
    },
    cancelForm() {
      this.loading = false;
      this.dialog = false;
      clearTimeout(this.timer);
    }
  }
}
</script>
```
[https://element-plus.gitee.io/#/zi-ding-yi-nei-rong](https://element-plus.gitee.io/#/zi-ding-yi-nei-rong)

![image](https://user-images.githubusercontent.com/22975974/100869025-67432a00-34d7-11eb-8878-951ae2052e72.png)

**Comment by JeremyWuuuuu** [comment]
Regression bug, will be fixed and released in beta.3

**Comment by JeremyWuuuuu** [comment]
What version are you using?
It seems fine on my machine with `beta.2`
![image](https://user-images.githubusercontent.com/15975785/100962304-08c18e80-355f-11eb-9948-28c4991f8235.png)

**Comment by lqsong** [comment]
> beta.2

`beta.2`

[https://element-plus.gitee.io/#/zh-CN/component/drawer](https://element-plus.gitee.io/#/zh-CN/component/drawer)

![image](https://user-images.githubusercontent.com/22975974/100962977-702c0e00-3560-11eb-862b-4f00b14c2c9c.png)

**Comment by JeremyWuuuuu** [comment]
This could be related to the rendering strategy, drawer always updates its `z-index` when `visibility` changes, the current documentation site has not been updated to the newest version I think. This should be fixed in next release.
Also I think we should suggest our user doing things with `how to avoid select component being covered by dialog like components`

**Comment by lqsong** [comment]
> This could be related to the rendering strategy, drawer always updates its `z-index` when `visibility` changes, the current documentation site has not been updated to the newest version I think. This should be fixed in next release.
> Also I think we should suggest our user doing things with `how to avoid select component being covered by dialog like components`


`1.0.1-beta.3` fix ,  `1.0.1-beta.4` again

**Comment by JeremyWuuuuu** [comment]
@lqsong They reason that this bug is showing and disappearing is all about the `insertion mode` of `el-popper` which in previous releases it was defaults to append into `document.body` and we have found some perf issue when it inserts to the body, then change it to `disabled` which means it won't insert to body by default then the perf should be better. Then we had to change it back because the style will be infected by the ancestors when you don't insert it to the `document.body`, so we had to change it back.
And the reason that `el-select` options was covered by `el-drawer` is because that `el-drawer` uses `dynamic z-index system` which was not enabled in `el-popper` (the foundation of `el-select-dropdown`, so I am going to fix this by enabling `dynamic z-index system` on `el-popper` this should not be a problem.
